src/utils/arrayEquality.ts

```diff
@@ -0,0 +1,11 @@
+export const arrayOfStringdMatch = (a?: string[], b?: string[]) => {
```

Comment:
Could you convert to a `Set` and use the set equality?

Reply:
I like this idea, I've not used `Set` much so far will round back to it later.